src/main/resources/hudson/plugins/jira/Messages.properties

```diff
@@ -36,4 +36,9 @@ SearchIssuesStep.Descriptor.DisplayName=JIRA: Search issues
 IssueSelectorStep.Descriptor.DisplayName=JIRA: Issue selector
 JiraCreateIssueNotifier.DisplayName=JIRA: Create issue
 IssueSelector.ExplicitIssueSelector.DisplayName=Explicit selector
-IssueSelector.JqlIssueSelector.DisplayName=JQL selector
\ No newline at end of file
+IssueSelector.JqlIssueSelector.DisplayName=JQL selector
+IssueUpdateStep.Descriptor.DisplayName=JIRA: Progress issues by workflow action
```

Comment:
Actually progress is a good word, hence i suggest "JIRA: Progress issues through workflow"